translator/trace/jaeger/jaegerthrift_to_traces.go

```diff
@@ -109,7 +109,7 @@ func jThriftSpanToInternal(span *jaeger.Span, dest pdata.Span) {
 	}
 
 	attrs := dest.Attributes()
-	attrs.InitEmptyWithCapacity(len(span.Tags))
+	attrs.EnsureCapacity(len(span.Tags))
```

Comment:
Here too.

Reply:
Not a problem, the output param is empty.